### PR TITLE
Cleaning up views and applying basic css styling

### PR DESCRIPTION
**Pull request by fogg4444**

### CODE REVIEW

File: gobbler/views/navBar.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template name='navBar'>
-  <div class='nav-bar'>
```
Comment (iandeboisblanc) [inline gobbler/views/navBar.html]: I noticed you took off the line with the class declaration here, but are still referencing it in the styling sheet. Is it classes elsewhere?

File: gobbler/views/welcome.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<template name="welcome">
+  {{> navBar }}
+  <div class="welcome">
+    <p>Welcome to Gobbler!</p>
+    <p>Remove the nav bar from this page after we build login event redirect.</p>
```
Comment (iandeboisblanc) [inline gobbler/views/welcome.html]: Just an FYI, the login redirect is currently commented out in the server.js file. I was having an issue where it was redirecting at random points. I didn't spend too long working on it though.

File: gobbler/views/main.html [hunk 3]
```diff
@@ -1,5 +1,13 @@
 <template name="main">
```
Comment (iandeboisblanc) [inline gobbler/views/main.html]: Can we move away from the template 'main'? I feel like it's not adequately descriptive. Maybe we should go with GibletManager or something? (Both in terms of file name and template name). Thoughts?

File: gobbler/views/navBar.html (hunk 1 above)
Comment (fogg4444) [inline gobbler/views/navBar.html]: nope, probably just forgot to remove it from the stylesheet.

File: gobbler/views/main.html (hunk 3 above)
Comment (fogg4444) [inline gobbler/views/main.html]: yeah, I called the file gibletView. The template name should be the same.

**Pull request closed without merging**